src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -163,7 +163,7 @@ public ResponseEntity patchObjection(
             List<AttachmentResponseDTO> attachmentResponseDTOs = attachments.stream()
                     .map(attachmentMapper::attachmentEntityToAttachmentResponseDTO).collect(Collectors.toList());
 
-            return responseEntityFactory.createResponse(ServiceResult.created(attachmentResponseDTOs));
+            return responseEntityFactory.createResponse(ServiceResult.found(attachmentResponseDTOs));
```

Comment:
does found return a 200? Wouldn't we want it to be OK?

Reply:
It return 200 checked with Tom Strickland and tested this.
<img width="1101" alt="Screen Shot 2020-07-14 at 12 29 27" src="https://user-images.githubusercontent.com/38655987/87420742-b8f12380-c5cd-11ea-924e-8fd076668574.png">